### ISSUE TITLE
Save apikey more securely

### ISSUE THREAD

**Issue by Freed-Wu** [wakatime/wakatime-cli#781]
> WakaTime plugins and wakatime-cli share a common INI config file:

> $WAKATIME_HOME/.wakatime.cfg

```
[settings]
debug         = false
hidefilenames = false
ignore        = """
    COMMIT_EDITMSG$
    PULLREQ_EDITMSG$
    MERGE_MSG$
    TAG_EDITMSG$"""
api_key       = XXXXXXXXXXXXXXXXXXXXXXXXX

[internal]
backoff_retries = 0
backoff_at      = 
```

**Describe the solution you'd like to see**

Why not save API tokens securely using [pass](https://www.passwordstore.org/) or [keyring](https://pypi.org/project/keyring/)?

**Describe alternatives you've considered**

```
[settings]
debug         = false
hidefilenames = false
ignore        = """
    COMMIT_EDITMSG$
    PULLREQ_EDITMSG$
    MERGE_MSG$
    TAG_EDITMSG$"""
api_key_cmd     = pass ls wakatime/api_key

[internal]
backoff_retries = 0
backoff_at      = 
```

**Comment by gandarez** [comment]
@Freed-Wu we've implemented `api_key_vault_cmd` option in `.wakatime.cfg` file to get any api key from any vault in `v1.56.0-alpha.1`. Depending on the IDE you work it won't take immediate effect before we start changing them.

**Comment by Freed-Wu** [comment]
Thanks. I wait <https://github.com/wakatime/vim-wakatime>, <https://github.com/wbingli/zsh-wakatime>, <https://github.com/irondoge/bash-wakatime>.

**Comment by gandarez** [comment]
It has been released on the stable `v1.57.0`. For `zsh`, `bash` and `vim` should work out of the box after you update to latest cli.
